specs/pvs/corla.pvs

```diff
@@ -1650,6 +1761,186 @@ END system_assumptions
+%%% overall, much of which is relevant to the State-wide dashboard. In
+%%% particular, geographic (such as the identity of the State and its
+%%% Counties) and political (registered political parties) information
+%%% must be stored, as must information about each election under
+%%% audit.
+%%%
+%%% Notice that these tables are simply a flattening of the election
+%%% datatype. This is a standard pattern in mapping HOL specifications
+%%% to relational data models.
+  geography_table: TYPE =
+    sequence[[nation, state, county]]
+  political_party_table: TYPE =
+    sequence[[political_party]]
+  national_contest_table: TYPE =
+    sequence[[nation, contest]]
+  state_contest_table: TYPE =
+    sequence[[state, contest]]
+  county_contest_table: TYPE =
+    sequence[[county, contest]]
+
+  public_meeting_to_determine_seed_table: TYPE =
+    [# location: string, date: string #]
+  rla_information_table: TYPE =
+    sequence[[RLA, audit_reason, contest, risk_limit]]
+  ballots_to_audit_list_table: TYPE =
+    sequence[[contest, sequence[ballot_manifest_info]]]
+
+%%% County administator actions create and update several kinds of
+%%% data. For example, they define who is on their audit boards, they
+%%% generate digests of files critical to the audit (principly, ballot
+%%% manifests and CVRs), they upload those files, etc.
+%%%
+%%% For general county information, we currently model the data as
+%%% property/value pairs in a record. For example, we imagine that
+%%% each county will want to display who their County Clerk is, etc.
+
+  county_general_information_table: TYPE =
+    sequence[[# property: string, value: string #]]
+  audit_board_table: TYPE =
+    sequence[[county, audit_board_member]]
+  ballot_manifest_digest_table: TYPE =
+    sequence[[# county: county,
+                ballot_manifest_file_name: string,
+                digest: string,
+                data: ballot_manifest_file #]]
+  cvr_digest_table: TYPE =
+    sequence[[# county: county,
+                cvr_file_name: string,
+                digest: string,
+                data: cvr_file #]]
+  ballot_manifest_table: TYPE =
+    sequence[[county, ballot_manifest_info]]
+
+%%% Cast vote records uploaded by counties contain CVRs of three
+%%% kinds: CVRs for local (county) contests, CVRs for contests that
+%%% span counties, and CVRs for contests that span the whole
+%%% state. These latter two cases are identical from a modeling point
+%%% of view, as the whole state is simply a spanning contest over all
+%%% counties.
+%%%
+%%% We can store and organize this information in several different
+%%% ways. First, we can simply store the individual CVRs uploaded by
+%%% a given county in a table.
+
+  cvr_table: TYPE =
+    sequence[[county, cvr]]
+
+%%% But we also need to store all CVRs for each contest, aggregating
+%%% all CVRs across the upload from all counties in which the contest
+%%% was on the ballot.
+
+  contest_cvr_table: TYPE =
+    sequence[[contest, cvr]]
+
```

Comment:
For the above two CVR tables, are we storing the individual CVRs redundantly in both tables, or is the idea that there is a single table containing CVRs and some ID assigned to each, which is then referenced in both these tables?